test/test_core.py

```diff
@@ -6652,7 +6653,10 @@ def test_cubescript(self, asyncify):
     if asyncify:
       if self.is_wasm64():
         self.skipTest('TODO: asyncify for wasm64')
-      self.set_setting('ASYNCIFY')
+      self.set_setting('ASYNCIFY', asyncify)
+    if asyncify == 2:
+      self.require_jspi()
+      self.emcc_args += ['-Wno-experimental']
```

Comment:
Should `require_jspi` add that flag to `emcc_args` perhaps? otherwise I think we'd need to repeat it manually in other tests?

Reply:
Yeah, I have another patch to clean up all the jspi tests and enable them for node canary and v8. I'll put it up after these few jspi tests merge.

Reply:
Sounds good!